src/util.py

```diff
@@ -17,7 +17,6 @@
 # The default length of the truncated hash used to identify parameter combinations
 DEFAULT_HASH_LENGTH = 7
 
-
```

Comment:
Was anything changed in this file?

Reply:
This file was somehow pulled into my branch, it can be removed